src/workerd/api/util.c++

```diff
@@ -146,7 +146,7 @@ kj::String redactUrl(kj::StringPtr url) {
   };
 
   for (const char& c: url) {
-    uint8_t lookup = kCharLookupTable[c];
+    uint8_t lookup = kCharLookupTable[static_cast<const kj::byte>(c)];
```

Comment:
Why do we need this? `const char` is an alias to `const kj::byte` isn't it?

Reply:
Ah I get it, char is not `uint8_t`

Reply:
`kj::byte` is an alias for `unsigned char`